optimum/intel/ipex/modeling_base.py

```diff
@@ -61,8 +61,8 @@
 _IPEX_SUPPORT_MODEL_TYPES = ("llama", "bert", "vit", "falcon", "gpt2", "qwen2")
 _IPEX_EXPORTED_GENERATION_METHODS = ("sample", "greedy_search", "beam_sample", "beam_search", "assisted_generation")
 _IPEX_MINIMUM_VERSION_FOR_COMPILE = "2.5.0"
-# TODO: Some models are already fixed in torch 2.6, will enable them when torch upgrading to 2.6
-_COMPILE_NOT_READY_MODEL_TYPES = ("electra", "roformer", "gpt_neox", "beit", "llama", "falcon", "gpt2", "qwen2")
+# Page attention model cannot use torch.compile for now.
+_COMPILE_NOT_READY_MODEL_TYPES = ("llama", "falcon", "gpt2", "qwen2")
```

Comment:
should we check the torch version and adapt `_COMPILE_NOT_READY_MODEL_TYPES` depending on it ?

Reply:
Not really, the ipex op is not compatible with torch.compile for now. We will remove it once ipex fixed this issue.

Reply:
my question is more : shouldn't `_COMPILE_NOT_READY_MODEL_TYPES` depend on the torch version ?

```python
if is_torch_version("<", "2.6"):
    _COMPILE_NOT_READY_MODEL_TYPES = ("electra", "roformer", "gpt_neox", "beit", "llama", "falcon", "gpt2", "qwen2")
else:
    _COMPILE_NOT_READY_MODEL_TYPES = ("llama", "falcon", "gpt2", "qwen2")
```

Reply:
thanks for adding in https://github.com/huggingface/optimum-intel/pull/1176/commits/af1536ffc9afc7f45439cf7eea1c2122fa2262be @jiqing-feng, merging now